cmake/modules/FindWaylandProtocolsWebOS.cmake

```diff
@@ -8,11 +8,11 @@
 # WAYLANDPROTOCOLSWEBOS_PROTOCOLSDIR -  directory containing the additional webOS Wayland protocols
 #                                       from the webos-wayland-extensions package
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_WAYLAND_WEBOS_SERVER wayland-webos-server>=1.0.0)
+  pkg_check_modules(PC_WAYLAND_WEBOS_SERVER wayland-webos-client>=1.0.0)
 endif()
 
 find_path(WAYLAND_PROTOCOLS_WEBOS_PROTOCOLDIR NAMES wayland-webos/webos-shell.xml
-                                              PATHS ${DEPENDS_PATH}/share NO_CACHE)   
+                                              PATHS ${DEPENDS_PATH}/share)   
```

Comment:
Doesn't this work for the depends buildsystem only? What if the protocols are installed on the host?

Reply:
@sundermann it does, that is something that needs fixing. Any suggestions?

Reply:
Forget about my comment. `DEPENDS_PATH` is `/home/mi/ssundermann/Sourcecode/buildroot-nc4/output/host/arm-webos-linux-gnueabi/sysroot/usr` so that should work if the protocols are built externally. However, I'd suggest using `find_file` instead because we are only interested in that one xml file.

Reply:
@sundermann Good idea, however we might need the other protocol XMLs in the future

Reply:
I agree, let's use `find_path` instead.